src/Kernel32.Shared/Kernel32+CreateFileFlags.cs

```diff
@@ -146,13 +145,13 @@ public enum CreateFileFlags : uint
             SECURITY_SQOS_PRESENT = 0x00100000,
 
             /// <summary>Impersonates a client at the Anonymous impersonation level.</summary>
-            SECURITY_ANONYMOUS = SECURITY_IMPERSONATION_LEVEL.SecurityAnonymous << 16,
+            SECURITY_ANONYMOUS = 0 << 16,
 
             /// <summary>The security tracking mode is dynamic. If this flag is not specified, the security tracking mode is static.</summary>
             SECURITY_CONTEXT_TRACKING = 0x00040000,
 
             /// <summary>Impersonates a client at the Delegation impersonation level.</summary>
-            SECURITY_DELEGATION = SECURITY_IMPERSONATION_LEVEL.SecurityDelegation << 16,
+            SECURITY_DELEGATION = 3 << 16,
```

Comment:
This seems odd though. We're not willing to declare the constants at this project, but we're willing to set the bits anyway? If the bits supported on non-desktop platforms, why can't we define the enum? And if the bits aren't supported, why are we using them?